src/api/app/jobs/stage_requests_job.rb

```diff
@@ -0,0 +1,28 @@
+class StageRequestsJob < ApplicationJob
+  def perform(staging_project, request_ids, user)
+    unassigned = []
+    User.current = user
+    requests = BsRequest.find(request_ids)
+    requests.each do |request|
+      action = request.bs_request_actions.first # we only care about the first one
+      if action.is_submit?
+        BranchPackage.new(
+          target_project: staging_project.name,
+          project: action.source_project,
+          package: action.source_package,
+          extend_package_names: false
+        ).branch
+      elsif request.is_delete?
+        # TODO: implement delete requests
+      end
+    # rubocop:disable Lint/RescueException
+    rescue Exception
```

Comment:
To get rid of the RuboCop disable line we could just catch ``ApiExceptions``?

Reply:
which one? https://github.com/vpereira/open-build-service/blob/d7d2e11f642cf8148573b5d12b5a607d4058c182/src/api/lib/api_exception.rb#L2

Reply:
I would catch all ``APIError`` as BranchPackage class is throwing ~10 different ones...